src/__experimental__/components/date/date.stories.js

```diff
@@ -67,7 +67,7 @@ function makeStory(name, themeSelector, component, disableChromatic = false) {
     themeSelector,
     info: {
       text: info,
-      propTables: [OriginalTextbox, DateInput],
+      propTables: [OriginalTextbox, BaseDateInput],
```

Comment:
I think this should just stay as `DateInput` as that is what the consumers are going to be importing